docs/resources/destination_mixpanel.md

```diff
@@ -97,6 +98,7 @@ Optional:
 - `track_categorized_pages` (Boolean) This will track events to Mixpanel for page method calls that have a category associated with them. For example page('Docs', 'Index') would translate to Viewed Docs Index Page.
 - `track_named_pages` (Boolean) This will track events to Mixpanel for page method calls that have a name associated with them. For example page('Signup') would translate to Viewed Signup Page.
 - `use_native_sdk` (Block List, Max: 1) Enable this setting to send the events via the device mode. (see [below for nested schema](#nestedblock--config--use_native_sdk))
+- `use_new_mapping` (Boolean) Use New Mapping
```

Comment:
should we add this recommendation as mentioned in UI?
<img width="489" alt="image" src="https://user-images.githubusercontent.com/9637254/180659336-40d466d4-bfa1-449b-be3b-590cd5698aeb.png">

Reply:
sure

Reply:
done @ktgowtham merging this.